File: MyWami/src/com/MyWami/model/TransmitModel.java
package com.MyWami.model;

/**
 * Created by robertlanter on 2/24/14.
 */
public class TransmitModel {
	private int fromIdentityProfileId;
	private int wamiToTransmitId;
	private String toProfileName;
	private String toEmailAddress;


	public String getToProfileName() {
		return toProfileName;
	}

	public void setToProfileName(String toProfileName) {
		this.toProfileName = toProfileName;
	}

	public String getToEmailAddress() {
		return toEmailAddress;
	}

	public void setToEmailAddress(String toEmailAddress) {
		this.toEmailAddress = toEmailAddress;
	}

	public int getFromIdentityProfileId() {
		return fromIdentityProfileId;
	}

	public void setFromIdentityProfileId(int fromIdentityProfileId) {
		this.fromIdentityProfileId = fromIdentityProfileId;
	}

	public int getWamiToTransmitId() {
		return wamiToTransmitId;
	}

	public void setWamiToTransmitId(int wamiToTransmitId) {
		this.wamiToTransmitId = wamiToTransmitId;
	}
}

File: MyWamiIOS/MyWami/MyWami/TransmitModel.swift
//
//  TransmitModel.swift
//  MyWami
//
//  Created by Robert Lanter on 4/16/15.
//  Copyright (c) 2015 Robert Lanter. All rights reserved.
//

import Foundation

public class Transmit {
    
    var fromIdentityProfileId: Int = 0
    var wamiToTransmitId: Int = 0
    var toProfileName: String = ""
    var toEmailAddress: String = ""
    
    func setFromIdentityProfileId(fromIdentityProfileId: Int) {
        self.fromIdentityProfileId = fromIdentityProfileId
    }
    
    func getFromIdentityProfileId() -> Int {
        return self.fromIdentityProfileId
    }
    
    func setWamiToTransmitId(wamiToTransmitId: Int) {
        self.wamiToTransmitId = wamiToTransmitId
    }
    
    func getWamiToTransmitId() -> Int {
        return self.wamiToTransmitId
    }
    
    func setToProfileName(toProfileName: String) {
        self.toProfileName = toProfileName
    }
    
    func getToProfileName() -> String {
        return self.toProfileName
    }

    func setToEmailAddress(toEmailAddress: String) {
        self.toEmailAddress = toEmailAddress
    }
    
    func getToEmailAddress() -> String {
        return self.toEmailAddress
    }

}